### PR TITLE
BCF-2665 Fix SQ regex for ci tests output

### PR DESCRIPTION
**Pull request by ilija42**
Sonar Qube can no longer find test coverage, because path to artifacts got changed. This should fix the SQ regex

### CODE REVIEW

File: .github/workflows/ci-core.yml [hunk 1]
```diff
@@ -109,7 +109,7 @@ jobs:
         if: always()
         uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
         with:
-          name: ${{ matrix.cmd }}_logs
+          name: go_core_tests_${{ matrix.cmd }}_logs
```
Comment (jmank88) [inline .github/workflows/ci-core.yml:112]: How does this work?
We already set `matrix.cmd` as:
```yaml
cmd: ["go_core_tests", "go_core_race_tests"]
```

Comment (ilija42) [reply .github/workflows/ci-core.yml:112]: Ok my bad I didn't see this But then I don't understand why SQ CI is broken, when the path is properly formatted

Comment (ilija42) [reply .github/workflows/ci-core.yml:112]: this was the path that got generated go_core/_race_tests_logs

Comment (jmank88) [reply .github/workflows/ci-core.yml:112]: :confused:

**Pull request merged**